Quadriga/src/main/java/edu/asu/spring/quadriga/transform/ResourceProvider.java

```diff
@@ -0,0 +1,111 @@
+package edu.asu.spring.quadriga.transform;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.core.runtime.Platform;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.emf.edit.domain.EditingDomain;
+import org.eclipse.osgi.service.datalocation.Location;
+
+public class ResourceProvider {
```

Comment:
remove this class. This will be replaced by your own classes that handle storing and retrieving files.